packages/lit-dev-content/site/blog/2023-10-10-lit-3.0.md

```diff
@@ -0,0 +1,175 @@
+
+Find them in their new npm homes:
+[`@lit/react`](https://www.npmjs.com/package/@lit/react),
+[`@lit/task`](https://www.npmjs.com/package/@lit/task), and
+[`@lit/context`](https://www.npmjs.com/package/@lit/context)
+
+These packages can now be considered stable, and have documentation and examples on lit.dev!
+
+Current users of these packages under the `@lit-labs` scope can migrate by first updating to the latest version of those packages to test for any breakages, then update the import to use the `@lit` scoped package. The latest `@lit-labs` scoped packages will receive the same updates until the next major version of the `@lit` scoped package.
+
+Thank you so much to everyone in the community who has tested out a Labs package, filed issues, discussed features, and led these packages to graduation! 🎓
+
+### React
+
+While custom elements can be used in React projects as is, there are some rough edges around their usage, namely:
+- Setting properties on elements (rather than attributes)
+- Adding handlers for custom events
+- Type checking custom elements and their props
+
+Some of these are being addressed by React in a future version, but they are currently only present in experimental builds not recommended for production.
+
+`@lit/react` allows creation of React components wrapping the custom element. The created components can be idiomatic to React such that users of the component do not have to worry about the inner web component implementaion.
+
+It is useful for both web component authors who wish to vend React versions of their components for users to reach a wider audience, as well as React developers who wish to use a neat web component that they found in their project more ergonomically.
+
+`@lit-labs/react` has been our most popular labs project by far with over 500k weekly npm downloads and is already being used by many web component libraries to provide React versions of their components to users.
+
+Read more about it at our [React framework integration](/docs/frameworks/react) documentation.
+
+### Task
+
+Rendering asynchronously retrieved data in a Lit component, while not terribly difficult to do, can involve a lot of boiler plate code to do properly with all the edge cases to handle.
+
+Some considerations include:
+- Conditionally rendering different content while the data is pending, retrieved, or the request failed
+- Handling race conditions, ensuring the data for the latest request is kept and previous requests are cancelled or ignored if completed later
+
+`@lit/task` provides a reactive controller to help manage this async workflow.
+
+We've found ourselves, on numerous occasions, starting a barebones example with asynchronous requests and quickly reaching to `@lit-labs/task` for its convenience. We've refined the API and added features like providing an `AbortSignal` while the project was being developed under labs. We are happy to officially graduate and encourage everyone to use it!
```

Comment:
```suggestion
We've found `@lit-labs/task` indispensable when working with asynchronous value. It includes a number of refinements since its initial release, like how every task receives an `AbortSignal` that the library ensures aborts if the task run becomes obsolete. 

We are happy for it to officially graduate and encourage everyone to use it! For more info see the new [Async Tasks](/docs/data/task/) docs page.
```

Reply:
"asynchronous value" -> "asynchronous values"